frontend/src/lib/components/PropertyFilters/components/PropertyValue.tsx

```diff
@@ -205,19 +205,10 @@ export function PropertyValue({
                 setInput('')
                 setValue('')
             }}
-            onChange={(val) => {
```

Comment:
There are 3 primary change events that happen with this autoComplete component - onSearch (user types into the box), onChange (user types into the box or selects an autocomplete suggestion) and onSelect (user selects autocomplete suggestion).

In all cases, onSearch and onSelect fire after onChange, and because react batches state updates, the state update in the onChange handler is useless, since the same state is updated in the onSearch and onSelect handlers.